src/documents/SubcomponentDeclaration.md

```diff
@@ -140,29 +155,64 @@ fluid.defaults("cspace.admin", {
 });
 ```
 
+### Injected Subcomponent Declaration
+
+Our second example shows a component being injected from one site in the tree to another. We call the standard, original
+component `concreteChild` a _concrete subcomponent_ because it is constructed in-place where it is defined. The site
+to which it is injected has a path `child2.injectedComponent` in the final instantiated tree, and at that site it is 
+called an _injected subcomponent_ because it appears at a path other than the one where it was constructed. In memory, the
+JavaScript object references at path `concreteChild` and `child2.injectedComponent` will be identical.
+
+<div class="infusion-docs-note"><strong>Note:</strong> You should use <strong>only</strong> this facility for
+shipping Infusion component references around within the tree, and not copy raw component references around by hand, otherwise you will
+confuse the framework's book-keeping for reference resolution.</div>
+
 ```javascript
-fluid.defaults("cspace.admin.showAddButton", {
-    gradeNames: ["fluid.modelComponent"],
-    components: { // injected component declaration
-        permissionsResolver: "{permissionsResolver}"
+fluid.defaults("examples.injection.root", {
```

Comment:
this code block and all subsequent documentation is all jumbled together in GitHub's markdown rendering. I'd assume that it's either caused by a malformed code block or something within the code block itself.